pkg/hoarder/discoverer.go

```diff
@@ -202,9 +212,7 @@ func (discoverer *CidDiscoverer) addProviderRecordsHttp(addProviderWG *sync.Wait
 					log.Errorf("error %s calling addAgentVersionToProvideStore", err)
 				} else {
 					log.Debug("Added agent version to provider store")
-				}
-				cidInfo.AddPublicationTime(trackableCid.PublicationTime)
-				cidInfo.AddProvideTime(trackableCid.ProvideTime)
+				} */
 
 				//TODO discoverer starting ping res
 				pingRes := models.NewPRPingResults(
```

Comment:
One extra thing here - I think that we could use the `tr.PublishTime` for the `PingResult.PingTime`

Reply:
```golang
// PRPingResults is the basic struct containing the result of the individual ping of a PR Holder.
type PRPingResults struct {
	Cid           cid.Cid
	PeerID        peer.ID
	Round         int
	FetchTime     time.Time
	FetchDuration time.Duration
	Active        bool
	HasRecords    bool
	ConError      string
}
```
Which of the PRPingResults struct's fields would that be?